files/usr/share/defaults/jeos-firstboot.conf

```diff
@@ -22,3 +22,9 @@
 # PXE / OEM) that are part of another product and than can be used
 # when this license was already accepted by other means.
 # JEOS_EULA_ALREADY_AGREED='yes'
+
+# If set to a nonempty value, the dialog box for showing the
+# SUSEConnect dialog help will not be displayed. By default this
+# dialog is present when SUSEConnect is installed on SLE systems, and
```

Comment:
I'd just drop the `, and requires user intervention to hide it.` - that's just normal dialog behaviour

Reply:
Yes, done.